automated_tests/docs/pktvisor/create_a_policy_using_admin_permission_with_all_handlers.md

```diff
@@ -0,0 +1,11 @@
+## Scenario: Create a policy using admin permission with all handlers 
+## Steps: 
+- Provide a pktvisor instance using `docker run --net=host -d ns1labs/pktvisor pktvisord --admin-api <net_interface>`
+- Create a policy with all handlers through a post request on the endpoint: `/api/v1/policies`
+  - Check our method `generate_pcap_policy_with_all_handlers` on ../../features/steps/policies.py in order to have examples of how to do it
```

Comment:
should add link instead of ."./../features/steps/policies.py"

Reply:
something like

`[policies.y](../../features/steps/policies.py) `

Reply:
good point